src/main.lua

```diff
@@ -1813,6 +1809,7 @@ end, function(msg)
 				Data = json.encode(tickResult.maybeDistributedEpoch),
 			})
 		end
+		print("Successfully ticked epoch " .. i)
```

Comment:
Should the reader expect this to mean `i` has been closed out? Or started? Might make more sense to use a verb other than "ticked".

Reply:
No - bc we may have just created the epoch and not distributed it yet

Reply:
removed this log as it is unclear. epochs can be in various states - the respective functions add logs about their behavior and outputs - https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f